marklogic-data-hub/src/test/resources/entity-manager-test/db-config.json

```diff
@@ -0,0 +1 @@
+{"path-namespace":[{"prefix":"es", "namespace-uri":"http://marklogic.com/entity-services"}], "range-path-index":[{"collation":"http://marklogic.com/collation/codepoint", "invalid-values":"reject", "path-expression":"//*:instance/Employee/name", "range-value-positions":false, "scalar-type":"string"}], "range-element-index":[{"collation":"http://marklogic.com/collation/en", "invalid-values":"reject", "localname":"decimal", "namespace-uri":null, "range-value-positions":false, "scalar-type":"string"}]}
```

Comment:
does this need to be all on one line?

Reply:
no. but I copied it from generated code and it came that way. wasn't worth my effort to format it.